pymeasure/instruments/advantest/advantestR624X.py

```diff
@@ -0,0 +1,2086 @@
+        values={False: 2, True: 1},
+        map_values=True
+    )
+
+    line_frequency = Instrument.setting(
+        "lf 0,%d",
+        """ Set the used power supply frequency (``LF``) to 50 or 60hz.
+        With this command, the integration time per PLC for the measurement
+        will be one cycle of the power supply frequency you are using.
+
+        :type: int
+        """,
+        validator=strict_discrete_set,
+        values={50: 1, 60: 2},
+        map_values=True
+    )
+
+    store_config = Instrument.setting(
+        "sav %d",
+        """ Set the memory area for the config to be stored at (``SAV``).
+        There are five memory areas from 0 to 4 for storing.
+
+        :type: int
+        """,
+        validator=strict_range,
+        values=range(0, 4),
+    )
+
+    load_config = Instrument.setting(
+        "rcl %d",
+        """ Set the memory area for the config to be loaded from (``RCL``).
+        There are five areas (0~4) where parameters can be loaded by the RCL command.
+
+        :type: int
+        """,
+        validator=strict_range,
+        values=range(0, 4),
+    )
+
+    def set_lo_common_connection_relay(self, enable, lo_relay=None):
+        """ Turn the connection relay on/off between the A channel
+        LO (internal analog common) and the LO (internal analog common)
+        of the B channel (``LTL``).
+
+        :param bool enable: A boolean property that controls whether or not the
+            connection relay is enabled. Valid values are True and False.
+        :param lo_relay: A boolean property that controls whether or not the internal
+            analog common relay is enabled. Valid values are True,
+            False and None (don't change lo relay setting).
+        :type lo_relay: bool, optional
+
+        """
+        enable = map_values(enable, {True: 2, False: 1})
+        lo_relay = map_values(lo_relay, {True: 2, False: 1, None: 3})
+
+        self.write(f'ltl 0,{enable},{lo_relay}')
+
+    def parse_measurement(self, measurement):
+        # TODO, convert header to object
+        if ' ' in measurement:    # split, and skip header
+            measurement = measurement.split(' ')
+            return (float(measurement[1]), measurement[0])
+        else:
+            return (float(measurement), None)
+
+    def read_measurement(self):
+        """ Reads the triggered value, for example triggered by the external input.
+        """
+        return self.parse_measurement(self.read())[0]
+
+
+class SMUChannel(Channel):
+
+    def __init__(self, parent, id, voltage_range, current_range):
+        super().__init__(parent, id)
+        self.voltage_range = voltage_range[id]
+        self.current_range = current_range[id]
+
+    def insert_id(self, command):
+        return command.format_map({self.placeholder: ord(self.id) - 64})
+
+    def clear_measurement_buffer(self):
+        """ Clears the measurement data buffer (``MBC``). """
+        self.write('mbc {{ch}}')
```

Comment:
Does that work with the double parenthesis?

If it were an f string, it would word, but not this way, I think.

Reply:
>Does that work with the double parenthesis?
Nice one!, it will not work. I'll change it.